[2025-01-27T16:08:17.408+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_price_pipeline_daily.start_producer_daily manual__2025-01-27T16:03:10.524834+00:00 [queued]>
[2025-01-27T16:08:17.414+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_price_pipeline_daily.start_producer_daily manual__2025-01-27T16:03:10.524834+00:00 [queued]>
[2025-01-27T16:08:17.415+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-01-27T16:08:17.432+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): start_producer_daily> on 2025-01-27 16:03:10.524834+00:00
[2025-01-27T16:08:17.435+0000] {standard_task_runner.py:57} INFO - Started process 101 to run task
[2025-01-27T16:08:17.438+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'stock_price_pipeline_daily', 'start_producer_daily', 'manual__2025-01-27T16:03:10.524834+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/stock_pipeline_dag.py', '--cfg-path', '/tmp/tmpou8wllke']
[2025-01-27T16:08:17.440+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask start_producer_daily
[2025-01-27T16:08:17.477+0000] {task_command.py:415} INFO - Running <TaskInstance: stock_price_pipeline_daily.start_producer_daily manual__2025-01-27T16:03:10.524834+00:00 [running]> on host 2f718d8427ce
[2025-01-27T16:08:17.905+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='stock_price_pipeline_daily' AIRFLOW_CTX_TASK_ID='start_producer_daily' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T16:03:10.524834+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-27T16:03:10.524834+00:00'
[2025-01-27T16:08:17.907+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-27T16:08:17.907+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec producer-daily python /opt/airflow/dags/producers/kafka_producer_daily.py']
[2025-01-27T16:08:17.914+0000] {subprocess.py:86} INFO - Output:
[2025-01-27T16:08:18.006+0000] {subprocess.py:93} INFO - python: can't open file '/opt/airflow/dags/producers/kafka_producer_daily.py': [Errno 2] No such file or directory
[2025-01-27T16:08:18.015+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-01-27T16:08:18.023+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-01-27T16:08:18.025+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=stock_price_pipeline_daily, task_id=start_producer_daily, execution_date=20250127T160310, start_date=20250127T160817, end_date=20250127T160818
[2025-01-27T16:08:18.045+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task start_producer_daily (Bash command failed. The command returned a non-zero exit code 2.; 101)
[2025-01-27T16:08:18.069+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-01-27T16:08:18.088+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
